pom.xml

```diff
@@ -3807,11 +3420,6 @@
         <artifactId>jetty-servlets</artifactId>
         <version>${version.org.eclipse.jetty}</version>
       </dependency>
-      <dependency>
-        <groupId>org.eclipse.jetty</groupId>
-        <artifactId>jetty-util</artifactId>
-        <version>${version.org.eclipse.jetty}</version>
```

Comment:
Why changing the alphabetical order?

Reply:
@yurloc first I removed the eclipse deps - saw that tests were failing - restored them again - and here I must disregard the alphabetical order - we can change this later on